quickstarts/hashicorp/hcp-consul/config.yml

```diff
@@ -0,0 +1,39 @@
+# Name of the quickstart (required)
+# sets the URL name of the quickstart on public I/O
+name: hcp-consul
+
+# Displayed in the UI (required)
+title: HCP Consul
+
+# Long-form description of the quickstart (required)
+description: |-
+  This Quickstart provides example configuration for HCP Consul reporting metrics into New Relic via New Relic's statsd integration.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |-
+  HCP Consul Quick Start sample statsd configuration
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic Labs
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - hcp
+  - hcp-consul
+  - hashicorp
+  - consul
+  - nrlabs
+  - nrlabs-data
+
+# Documentation references
+documentation:
+  - name: Configuration documentation
+    url: https://github.com/newrelic-experimental/hashicorp-quickstart-annex/tree/main/hcp-consul/README.md
```

Comment:
I'm getting a 404 from this link

Reply:
@aswanson-nr Repo is fixed, sorry about that.